src/resampler.rs

```diff
@@ -0,0 +1,279 @@
+// License: MIT Copyright © 2024 Frequenz Energy-as-a-Service GmbH
```

Comment:
We already have a lot of code that does it like this, so just for consistency:
```suggestion
// License: MIT
// Copyright © 2024 Frequenz Energy-as-a-Service GmbH
```